doc/modules/preprocessing.rst

```diff
@@ -459,6 +459,8 @@ K-bins discretization
   >>> est.bin_width_
   array([ 3.,  1.,  2.])
 
+By default the output is one-hot encoded into a sparse matrix (See :class:`OneHotEncoder`)
```

Comment:
I'd rather this referred to a section of the user guide which described what this *means* rather than provides a (in this context irrelevant) tool to do it.

Reply:
@jnothman The only place I can think of is http://scikit-learn.org/stable/modules/preprocessing.html#encoding-categorical-features. Could you please provide more specific suggestion? Thanks.

Reply:
That seems the right place to point to.